manage.py

```diff
@@ -1,280 +1,10 @@
-from datetime import datetime
-
+from listenbrainz.manage import cli
 import listenbrainz.db.dump_manager as dump_manager
 import listenbrainz.spark.request_manage as spark_request_manage
-from listenbrainz.listenstore.timescale_utils import recalculate_all_user_data as ts_recalculate_all_user_data, \
```

Comment:
sphinx-click has a bug due to which it errors while trying to generate docs for multicommand groups. Therefore, I have moved actual commands to listenbrainz/manage.py and kept add dump and spark commands in root manage.py.